tests/labhub_test.py

```diff
@@ -292,5 +292,7 @@ def test_invite_me(self):
         plugins.labhub.os.environ['GH_TOKEN'] = 'patched?'
         testbot.assertCommand('!invite me',
                               'We\'ve just sent you an invite')
+        testbot.assertCommand('!hey there invite me',
+                              '', timeout=100)
```

Comment:
1. Move this change below line #298
2. `Test it against `Command not found`` whatever the string was produced when the tests failed.
3. Assert that there are no other messages in the queue, by appending the following after the assert statement

```py
         with self.assertRaises(queue.Empty):
             testbot.pop_message()
```